backend/planty/dashboard/models.py

```diff
@@ -35,3 +35,19 @@ class Plant(models.Model):
 
     class Meta:
         app_label = 'dashboard'
+
+    def values(self):
```

Comment:
Widziałem, że to się robi jakoś za pomocą serializer'ów, ale nie szukałem jak dokładnie ma to wyglądać. Póki co może być tak (może tak i zostanie :smile: )